File: kotlin/src/main/kotlin/cl/payments/constants/Providers.kt
package cl.payments.constants

import cl.payments.types.PaymentProvider

// Lista de proveedores soportados
val SUPPORTED_PROVIDERS = listOf(
    PaymentProvider.WEBPAY,
    PaymentProvider.GETNET
)

// Nombres legibles de proveedores
val PROVIDER_NAMES = mapOf(
    PaymentProvider.WEBPAY to "Webpay (Transbank)",
    PaymentProvider.GETNET to "Getnet (Santander)"
)

File: src/constants/providers.ts
import { PaymentProvider } from '../types';

// Lista de proveedores soportados
export const SUPPORTED_PROVIDERS: PaymentProvider[] = ['webpay', 'getnet'];

// Nombres legibles de proveedores
export const PROVIDER_NAMES: Record<PaymentProvider, string> = {
  webpay: 'Webpay (Transbank)',
  getnet: 'Getnet (Santander)',
};